Libraries/LibKeyboard/CharacterMap.h

```diff
@@ -43,9 +43,13 @@ class CharacterMap {
 
     u32 get_char(KeyEvent);
     void set_character_map_data(CharacterMapData character_map_data);
+    void set_character_map_name(String& character_map_name);
+
+    String character_map_name();
```

Comment:
This should be `const` qualified, as it does not mutate the state of the `CharacterMap`.